mongo/client_encryption.go

```diff
@@ -105,6 +105,17 @@ func (ce *ClientEncryption) Encrypt(ctx context.Context, val bson.RawValue, opts
 		transformed.SetKeyAltName(*eo.KeyAltName)
 	}
 	transformed.SetAlgorithm(eo.Algorithm)
+	if eo.QueryType != nil {
+		switch *eo.QueryType {
+		case options.QueryTypeEquality:
+			val := cryptOpts.QueryTypeEquality
+			transformed.SetQueryType(&val)
+		}
```

Comment:
Should we add a `default` case that returns an error if someone passes in an unknown query type?

Reply:
Good idea. Done.